src/plugins/CaptionsPlugin.js

```diff
@@ -76,11 +227,21 @@ export class CaptionsPlugin extends ButtonPlugin {
         for (let i = 0; i < this.captionsButtonLength; i ++) {
           this._captionsButtons[i].displayButton($event.data);
         }
+        for (let i = 0; i < this.colorRadiosLength; i++) {
```

Comment:
Idea:

Create a getter for all radios:

```javascript
get radios() {
  return this.colorRadios
    .concat(this.alignmentRadios)
    .concat(this.fontSizeRadios);
}
```

Then use that for loops:

```javascript
for (const radio of this.radios) {
  radio.displayRadios($event.data);
}
```

Reply:
Oh I like that idea!